nodejs/awsx/elasticloadbalancingv2/application.ts

```diff
@@ -122,12 +122,15 @@ export class ApplicationTargetGroup extends mod.TargetGroup {
     }
 
     public createListener(name: string, args: ApplicationListenerArgs,
-                          opts?: pulumi.ComponentResourceOptions): ApplicationListener {
+                          opts: pulumi.ComponentResourceOptions = {}): ApplicationListener {
+        // We didn't use to parent the listener to the target group.  Now we do.  Create an alias
+        // from the old parent to the current one if this moves over.
+        const aliases = [pulumi.createUrn(name, "awsx:x:elasticloadbalancingv2:ApplicationListener", opts.parent)];
```

Comment:
Technically - I think we need to merge this new alias into any aliases already provided in `opts`?  If the user provides any aliases, the code below will overwrite `aliases` with their array instead of this array - right?

Reply:
You're correct.  Have updated accordingly.